pkg/crosscluster/logical/logical_replication_job.go

```diff
@@ -887,14 +890,50 @@ func (r *logicalReplicationResumer) ingestWithRetries(
 	ctx context.Context, execCtx sql.JobExecContext,
 ) error {
 	ingestionJob := r.job
+	details := ingestionJob.Details().(jobspb.LogicalReplicationDetails)
 	ro := getRetryPolicy(execCtx.ExecCfg().StreamingTestingKnobs)
 	var err error
 	var lastReplicatedTime hlc.Timestamp
+	resolvedDest, err := resolveDest(ctx, execCtx.ExecCfg(), details.SourceClusterConnUri)
+	if err != nil {
+		log.Warningf(ctx, "failed to resolve destination details for change monitoring: %v", err)
+	}
+
 	for retrier := retry.Start(ro); retrier.Next(); {
-		err = r.ingest(ctx, execCtx)
+		confPoller := make(chan struct{})
+		g := ctxgroup.WithContext(ctx)
+		g.GoCtx(func(ctx context.Context) error {
+			defer close(confPoller)
```

Comment:
we usually add pollers that run during the LDR distsql flow here: https://github.com/edwardguo-crl/cockroach/blob/alter-connection-pcr/pkg/crosscluster/logical/logical_replication_job.go#L337

Could you refactor this to follow this convention?